Util_Widget.php

```diff
@@ -100,7 +100,7 @@ static public function add( $widget_id, $widget_name, $callback,
 		if ( $control_callback && current_user_can( 'edit_dashboard' ) &&
 			is_callable( $control_callback ) ) {
 			$w3tc_dashboard_control_callbacks[$widget_id] = $control_callback;
-			if ( isset( $_GET['edit'] ) && $widget_id == $_GET['edit'] ) {
+			if ( isset( Util_Request::get_string( 'edit' ) ) && $widget_id == Util_Request::get_string( 'edit' ) ) {
```

Comment:
Use `! empty()`.  You may want to set a variable to avoid calling twice.

Reply:
Changed to ! empty and created a variable to avoid 2 function calls.